cobra/tests/testbasic.py

```diff
@@ -110,6 +112,65 @@ def test_cobra_refcount(self):
 
     #def test_cobra_ssl(self):
     #def test_cobra_ssl_clientcert(self):
+    def test_cobra_helpers(self):
```

Comment:
An extremely minor nit, but couldn't you clean this up with just a for loop on the portnum and not just a while true?

Reply:
done